src/main/scala/com/databricks/spark/xml/functions.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019 Databricks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.databricks.spark.xml
+
+import org.apache.spark.annotation.Experimental
+import org.apache.spark.sql.{Column, Dataset}
+import org.apache.spark.sql.catalyst.parser.CatalystSqlParser
+import org.apache.spark.sql.types.StructType
+
+import com.databricks.spark.xml.util.InferSchema
+
+/**
+ * Support functions for working with XML columns directly.
+ */
+object functions {
```

Comment:
Sean, is the style rule change because of this? I think we could just ignore this for now (as Spark side does):

```scala
// scalastyle:off
object functions {
// scalastyle:on
```

Reply:
I figured, this is the only object the style rule covers, so, if it doesn't match the rule, the rule can be relaxed? it really won't matter either way